generative_adversarial_networks/restyle-encoder/restyle-encoder.py

```diff
@@ -0,0 +1,387 @@
+import os
+import sys
+import subprocess
+import time
+
+import numpy as np
+import cv2
+import onnx
```

Comment:
Load onnx and onnx runtime only if the onnx_runtime option is specified.

Reply:
Done.